.circleci/config.yml

```
@@ -86,6 +87,127 @@ jobs:
           paths:
             - fv3core/test_data
 
+  savepoints_mpi_54rank:
+    machine:
+      image: ubuntu-2004:202111-02
+      resource_class: xlarge
+    parameters:
+      backend:
+        description: "gt4py backend"
+        default: "numpy"
+        type: enum
+        enum: ["numpy"]
+    environment:
+      GOOGLE_APPLICATION_CREDENTIALS: /tmp/key.json
+    steps:
+      - run: |
+          sudo apt-get update && sudo apt-get install -y make \
+            software-properties-common \
+            libopenmpi3 \
+            libopenmpi-dev \
+            libboost-all-dev \
+            python3 \
+            python3-pip \
+      - gcp-cli/install:
+        version: 323.0.0
+      - run:
+          name: gcloud auth
+          command: |
+            echo $ENCODED_GCR_KEY
+            echo $ENCODED_GCR_KEY | base64 -d > $GOOGLE_APPLICATION_CREDENTIALS
+            gcloud auth activate-service-account --key-file=$GOOGLE_APPLICATION_CREDENTIALS
+            gcloud auth configure-docker
+      - checkout
+      - run:
+          name: Update Submodules
+          command: git submodule update --init
+      - run:
+          name: save gt4py_version.txt
+          command: git submodule status external/gt4py | awk '{print $1;}' > gt4py_version.txt
+      - restore_cache:
+          keys:
+            - v1-gt_cache_54ranks-<<parameters.backend>>-{{ checksum "gt4py_version.txt" }}
+            - v2-savepoints-c12_54ranks_standard-{{ checksum "Makefile.data_download" }}
+            - v2-pip_cache-{{ checksum "requirements_dev.txt" }}-{{ checksum "constraints.txt" }}
+      - run:
+          name: set up environment
+          command: |
+            pip3 install --upgrade setuptools wheel
+            pip3 install -r requirements_dev.txt -c constraints.txt
+      - save_cache:
+          key: v2-pip_cache-{{ checksum "requirements_dev.txt" }}-{{ checksum "constraints.txt" }}
+          paths:
+            - /home/circleci/.cache/pip
+      - run:
+          name: run tests
+          command: |
+            TEST_ARGS="--backend=<<parameters.backend>> -v -s" CONTAINER_CMD="" NUM_RANKS=54 EXPERIMENT="c12_54ranks_standard" FV3_PATH=$(pwd)/fv3core make savepoint_tests_mpi
+          no_output_timeout: 3h
+      - save_cache:
+          key: v1-gt_cache_54ranks-<<parameters.backend>>-{{ checksum "gt4py_version.txt" }}
+          paths:
+            - .gt_cache
```

Something like this _should_ in theory work.

```suggestion
      - run:
          name: run tests
          command: |
            GT_CACHE_DIR_ROOT=".pace_gtcache" TEST_ARGS="--backend=<<parameters.backend>> -v -s" CONTAINER_CMD="" NUM_RANKS=54 EXPERIMENT="c12_54ranks_standard" FV3_PATH=$(pwd)/fv3core make savepoint_tests_mpi
          no_output_timeout: 3h
      - save_cache:
          key: v1-gt_cache_54ranks-<<parameters.backend>>-{{ checksum "gt4py_version.txt" }}
          paths:
            - .pace_gtcache
```

---

Updated with something like this.